onto/clinical.py

```diff
@@ -0,0 +1,69 @@
+# ***automatically_generated***
+# ***source json:forte/ontology_specs/medical.json***
+# flake8: noqa
+# mypy: ignore-errors
+# pylint: skip-file
+"""
+Automatically generated ontology medical. Do not change manually.
+"""
+
+from dataclasses import dataclass
+from forte.data.data_pack import DataPack
+from forte.data.ontology.core import FList
+from forte.data.ontology.top import Generics
+from ft.onto.base_ontology import EntityMention
+from typing import List
+from typing import Optional
+
+__all__ = [
+    "UMLSConceptLink",
+    "MedicalEntityMention",
+]
+
+
+@dataclass
+class UMLSConceptLink(Generics):
```

Comment:
Let's use the classes here instead of copying: https://github.com/asyml/forte/blob/master/ftx/onto/clinical.py